justfile

```diff
@@ -0,0 +1,20 @@
+default:
+  @just --list
+
+create-control-plane:
+  kind create cluster --config ./kubernetes/kind-cluster-config.yml
+
+deploy-ingress-controller:
+  kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
+
+wait-for-ingress-controller:
+  kubectl wait --namespace ingress-nginx --for=condition=ready pod --selector=app.kubernetes.io/component=controller --timeout=600s
+
+deploy-astria:
+  kubectl apply -k kubernetes/
+
+wait-for-astria:
```

Comment:
I got an error when running this (and also same for the `kubectl get -n astria-dev-cluster` commands in the README):
```
$ just wait-for-astria
kubectl wait -n astria-dev-cluster deployment astria-dev-cluster-deployment --for=condition=Available=True
Error from server (NotFound): namespaces "astria-dev-cluster" not found
error: Recipe `wait-for-astria` failed on line 17 with exit code 1
```

Reply:
@noot did you run `just deploy-astria` before waiting for it? If yes, can you please provide `kubectl get namespaces`?

Reply:
It's because I pushed the temporary local image names I was using